nservicebus/msmq/scaling-out_overview_core_6.partial.md

```diff
@@ -16,4 +16,6 @@ Following table compares both approaches:
 | Round-robin load balancing                  | Fair load balancing                         |
 
 
-WARNING: When using sender-side distribution scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
\ No newline at end of file
+WARNING: When using sender-side distribution scaling out technique in a mixed version environment, make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
```

Comment:
@SzymonPobiega - I want to highlight the main concern first because it is a warning. However I get what you;re saying as well. The original sentence was too long and i tried to break it apart for better reading. 

How about this instead:

WARNING: When an endpoint uses the Sender-Side distribution feature to distribute commands or events to other endpoints running Versions 5 and below, it is possible that some or all of these receiving endpoints can process the same message.  To avoid duplicate messages from being processed, it is important that a distributor is deployed in front of the endpoint that is using Sender-Side distribution. For more details, refer to the [distributor sample.](/samples/scaleout/distributor/) This restriction does not apply if the endpoint that's using this feature and all other participating endpoints are on Versions 6 and above.